app/cafe.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+import app.errors as error
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise error.NotVaccinatedError
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise error.OutdatedVaccineError
+        if visitor["wearing_a_mask"] is False:
+            raise error.NotWearingMaskError
+        else:
```

Comment:
remove unnecessary else

Reply:
How can I add that without else?
![Screenshot (85)](https://user-images.githubusercontent.com/110389820/194357147-cf62247e-a769-4794-9a98-dd277b73ea71.png)